fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -229,6 +248,45 @@ public void setIsAllowNull(boolean isAllowNull) {
         this.isAllowNull = isAllowNull;
     }
 
+    public String getMetaDefaultValue(List<String> extras) {
+        String defaultValue = FeConstants.null_string;
+        if (this.defaultValue != null) {
+            return defaultValue;
+        } else if (defaultExpr != null) {
+            if ("now()".equalsIgnoreCase(defaultExpr.getExpr())) {
+                defaultValue = "CURRENT_TIMESTAMP";
+                extras.add("DEFAULT_GENERATED");
+            }
+        }
+        return defaultValue;
+    }
+
+    // if the value or expr calculated is the same for each row of a batch like now(). return true
+    // else for a batch of every row different like uuid(). return null
+    public boolean hasDefaultValue() {
+        if (defaultValue != null) {
+            return true;
+        }
+        if (defaultExpr != null && "now()".equalsIgnoreCase(defaultExpr.getExpr())) {
```

Comment:
Better to create an private function to encapsulate this logic.
Such as hasDefaultExpr